parsers/nmparser/parse_lst_file_test.go

```diff
@@ -9,8 +9,8 @@ import (
 )
 
 func TestParseOBJV(t *testing.T) {
```

Comment:
I'm fine with adding this test (assuming it passes) but these unit tests are largely broken and don't get run in CI, etc. for that reason.

The quasi-integration tests [here](https://github.com/metrumresearchgroup/babylontest) are what we really rely on. I would advocate adding another test model in [here](https://github.com/metrumresearchgroup/babylontest/tree/master/testdata/bbi_summary) and adding it to [the happy path list](https://github.com/metrumresearchgroup/babylontest/blob/master/bbi_summary_test.go#L28) in `bbi_summary_test.go`. Even if we don't actually write any new tests, that will at least cover this situation broadly.

@dpastoor @timwaterhouse if you can get me either a fake or real control stream for this (ideally with an output dir already populated, i.e. already run) then I can make the PR in babylontest for us.